lib/Dancer/Plugin/SimpleCRUD.pm

```diff
@@ -1127,13 +1164,25 @@ SEARCHFORM
         ),
         @foreign_cols,    # already assembled from quoted identifiers
         @custom_cols,
+        @join_cols
     );
     my $add_actions
         = $args->{editable}
         ? ", $table_name.$key_column AS actions"
         : '';
     my $query = "SELECT $col_list $add_actions FROM $table_name";
+    if (my $joins = $args->{joins}) {
+        for my $joiner (@$joins) {
+            my $join_style = $joiner->{join_style} || "join";
+            my $join_table = $joiner->{db_table} || die "'joins' directive needs 'db_table' setting";
```

Comment:
I'd like to see quote_identifier() around this.  I'd also like the doco to show that join_style is interpolated right into SQL, so to be careful - or, perhaps  better, require it to be only one of the recognised join types.